lib/uktest/tests.ld

```diff
@@ -9,4 +9,4 @@ SECTIONS
 		uk_asserttab_end = .;
 	}
 }
-INSERT AFTER .data;
+INSERT AFTER .rodata;
```

Comment:
The test sections have to be RW so they still have to be located after `.data` 😃

Reply:
Ah. I forget that the test results are stored in the section...